vendor/knative.dev/reconciler-test/pkg/feature/feature.go

```diff
@@ -258,6 +258,19 @@ func DeleteResources(ctx context.Context, t T, refs []corev1.ObjectReference) er
 		return true, nil
 	})
 	if err != nil {
+		gv, _ := schema.ParseGroupVersion("eventing.knative.dev/v1alpha1")
```

Comment:
Group is
https://github.com/knative-extensions/eventing-kafka-broker/blob/828b81ede7af938cdffd01f6f103992469330728/control-plane/config/eventing-kafka-broker/100-kafka-internal/100-consumergroup.yaml#L24

```suggestion
		gv, _ := schema.ParseGroupVersion("internal.kafka.eventing.knative.dev/v1alpha1")
```